src/main/java/org/checkerframework/checker/dependencyinjection/utils/KnownBindingsValue.java

```diff
@@ -0,0 +1,49 @@
+package org.checkerframework.checker.dependencyinjection.utils;
+
+/** A class that represents another class that is known to be bound in a Guice module. */
+public class KnownBindingsValue {
```

Comment:
I'd use Lombok's `@Builder` or AutoValue rather than writing this builder myself.

Also, you should turn on the Called Methods Checker in the build.gradle file.